lib/util/common.ts

```diff
@@ -59,3 +61,28 @@ export function detectPlatform(
 
   return null;
 }
+
+export function parseJson(content: string, filename: string): unknown {
+  return filename.endsWith('.json5')
+    ? JSON5.parse(content)
+    : parseJsonWithFallback(content, filename);
+}
+
+export function parseJsonWithFallback(
+  content: string,
+  context: string
```

Comment:
```suggestion
  filename: string
```
isn't that a better name? 🤔

Reply:
context could be something other as filename as well. We are using filename for now. Maybe will use something different in future.

Reply:
https://github.com/renovatebot/renovate/pull/24612#discussion_r1338249144